src/ast.mli

```diff
@@ -76,18 +70,37 @@ and stmt =
   | Break
   | Continue
 
-type func = {
-  name : string;
-  return_t : types;
-  formals : formal list;
-  body : stmt list;
+
+type message = {
+    name: string;
+    formals: formal list;
+}
```

Comment:
I think we discussed this with @aethos last weekend. But now I cannot recall why we thought it's OK to remove this.

`sender` can just be a `pid` right? Do you remember what we arrived at @aethos ?